boards/arc/nsim/nsim_sem_mpu_stack_guard.yaml

```diff
@@ -1,5 +1,5 @@
 identifier: nsim_sem_mpu_stack_guard
-name: SEM Nsim simulator
+name: SEM Nsim simulator (stack guard)
```

Comment:
while you're at it, Nsim should be nSIM

Reply:
Thank you for your feedback. I've applied changes accordingly.